taxonomies/categories.txt

```diff
@@ -39432,12 +39448,10 @@ ciqual_food_name:en:Black mulberry, raw
 ciqual_food_name:fr:Mûre noire (du mûrier), crue
 
 <en:Blackberries
-fr:Mûres fraîches
+<en:Fresh fruits, Raw blackberries
+en:Fresh blackberries
+fr:Mûres fraîches, Mûres de ronce crues
```

Comment:
Shouldn't "Mûres de ronce crues" be a synonym of the parrent "Raw blackberries"? Or do we assume "Mûres de ronces crues" are never frozen?